tests/snapshots/tests.contrib.anthropic.test_anthropic.test_anthropic_llm_multiple_prompts_no_history.json

```diff
@@ -0,0 +1,41 @@
+[[
```

Comment:
Is this snapshot needed? Don't see it used anywhere

Reply:
removed